src/config.js

```diff
@@ -1,51 +1,87 @@
 var chalk = require('chalk');
 
+// Produce overrided configuration
 function getChalkColors(defaultConfig, overrideConfig) {
-	// make the read file understandable to chalk.
-	var chalkColors = buildChalkFunction(defaultConfig);
-	var overrideChalkColors = buildChalkFunction(overrideConfig);
-	return Object.assign({}, chalkColors, overrideChalkColors);
+	var effectiveConfig = Object.assign({}, defaultConfig, overrideConfig);
+
+	// make the effectiveConfig understandable to chalk
+	// and return it
+	return buildChalkFunction(effectiveConfig);
 }
 
 function buildChalkFunction(config) {
 	var chalkColors = {};
-	for (var color in config.colors) {
-		if (config.colors.hasOwnProperty(color)) {
+	for (var outputMethod in config) {
+		if (config.hasOwnProperty(outputMethod)) {
 			var colorFunc = chalk;
 			//build up the chalk function
 			try {
-				config.colors[color].forEach(function(style) {
+				config[outputMethod].forEach(function (style) {
 					colorFunc = colorFunc[style];
 				});
 				if (typeof colorFunc === 'undefined') {
 					throw new Error('Unsupported colorFunc');
 				}
-				chalkColors[color] = colorFunc;
+				chalkColors[outputMethod] = colorFunc;
 			} catch (err) {
-				console.log('Error setting style for ' + color + '. Will use default style.');
+				console.log('Error setting style for ' + outputMethod + '. Will use default style.');
 			}
 		}
 	}
+	// console.log(chalkColors.default("fsd"));
 	return chalkColors;
 }
 
+// require default configuration
 var configInput = require('../lmeDefaultConfig.json');
 var providedConfig = {};
 try {
 	// try to get the config file from root file of the application.
 	providedConfig = require('app-root-path').require('lmeconfig.json');
+
+	// mock provided config: comment out this block before releasing it to production.

```

Comment:
need to remove some of these unused code blocks.. (this and console.logs)...makes it look like messy code